Ontology/DTDLv2/RealEstateCore/Space/Space.json

```diff
@@ -54,7 +54,7 @@
     {
       "@type": "Property",
       "description": {
-        "en": "A GeoJSON Polygon coordinate listing representing the geometrical representation of the space. Coordinates may be expressed in two or three dimensions. Ex: [[30.0, 10.0, 0.0], [40.0, 40.0, 2.0], [20.0, 40.0, 2.0], [10.0, 20.0, 2.0], [30.0, 10.0, 0.0]]."
+        "en": "A GeoJSON Polygon coordinate listing representing the geometrical representation of the space. Coordinates may be expressed in two or three dimensions. Ex: [[30.0, 10.0, 0.0], [40.0, 40.0, 2.0], [20.0, 40.0, 2.0], [10.0, 20.0, 2.0], [30.0, 10.0, 0.0]]. Unit: meters."
```

Comment:
Thanks, good comments!

The reason GeoJSON is in, is its relative simplicity and the plethora of tools for consuming/generating it. We want as low a barrier-to-entry as possible for devs, without providing "footguns" that lead them to make obvious mistakes. 

1. GeoJSON also allows for eastings and northings (https://www.rfc-editor.org/rfc/rfc7946#section-3.1.1, after the first bullet list) which we don't find in WGS84, and which are typically expressed in some culturally relevant distance unit. The spec also mentions pre-existing agreements on alternate representations (https://www.rfc-editor.org/rfc/rfc7946#section-4, second paragraph). We were hoping to lean on these parts when requiring local (x,y)-coordinates in meters.
2. My quick and dirty (faulty) assumption from https://www.rfc-editor.org/rfc/rfc7946#section-3.1.1 was that since a third element in coordinates is allowed, that ought to be enough to represent 3d geometries. But it is the ordering of those coordinates in an array that forms the (2D) polygon; and in a 3D setting, that doesn't work in the same way. Thanks for pointing this out, great to have your expertise here! 
3. Might one instead interpret GeoJSON multipolygons as 3D shapes, each child polygon defining one outside "edge" of the 3D shape?

Reply:
For 1: Ah, yes using a local coordinate system makes sense here.

For 3: As long as it's documented, I suppose multipolygons could be used to define even a complex 3D shape.